raco/myrial/interpreter.py

```diff
@@ -145,7 +145,7 @@ def empty(_scheme):
 
     def select(self, args):
         """Evaluate a select-from-where expression."""
-        op = self.bagcomp(args.from_, args.where, args.select)
+        op = self.bagcomp(args.from_, args.where, None, args.select)
```

Comment:
Don't we want to allow `ORDER BY` in SQL `SELECT` statements, not just MyriaL bag comprehensions?